### PR TITLE
Add PGConfEU blog

### PR DESCRIPTION
**Pull request by gulcin**
## Summary

-

### Blog Release Checklist

- \[x] Release on: \<mm/dd/yy>
- \[ ] Published on dev.to (connected to Xata org)
- \[x] OG image included
- \[ ] Social media is scheduled for this post

### CODE REVIEW

**Review by simonapencea — APPROVED** [review]

File: pgconfeu-2024.mdx [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+
+I am a firm believer of investing in diversity and inclusion initiatives. And it is usually a long-term investment. First, you need to bring people in, invite them to conferences,
+support them with travel funds, donate conference tickets, make sure they will feel safe and welcome when they join the conferences and eventually feel confident to give talks.
+After that, some of them will become active contributors to the community, some will be interested in helping with those conferences, some will become Postgres committers.
+
+Having a community that is inclusive makes things better for everyone, not only for the underrepresented groups.
+For this reason, I've recently joined newly forming [Postgres Europe Diversity Committee](https://www.postgresql.eu/diversity/) and we will have our first meeting in Athens on 22nd October.
+We have a lot of plans and initiatives to make the Postgres community more inclusive and diverse and I am happy to be part of it.
+
+### Extension Ecosystem Summit
```
Comment (CezzaineZ) [inline pgconfeu-2024.mdx:125]:
```suggestion
### <span id="ecosystem-summit">Extension Ecosystem Summit</span>
```

File: pgconfeu-2024.mdx [hunk 2]
```diff
@@ -0,0 +1,309 @@
...
+    'berlin',
+    'lisbon',
+    'tallinn',
+    'fosdem'
+  ]
+published: true
+slug: pgconfeu-2024
+ogImage: https://us-east-1.storage.xata.sh/uq2d57de9bli5qd91j16pjps7pfto8vk
+---
+
```
Comment (CezzaineZ) [inline pgconfeu-2024.mdx:48]:
```suggestion
As [PGConf.EU](https://pgconf.eu/) kicks off this week in Athens, I wanted to share why this event is special to me and thanks to Xata I'm able to once again attend and be involved in many initiatives. 

You can find me talking at the [Ecosystem Summit](#ecosystem-summit), chairing the Postgres Europe Diversity Committee, or hosting a Xata dinner (grab your RSVP below). But also if you're planning to attend the conference, don't miss out on this talk: Postgres schema migrations using the expand/contract pattern which is hosted by pgroll creator / Xata Software Engineer, Andrew Farries.

But first, a little bit of history...
```

**Review by CezzaineZ — COMMENTED** [review]
@gulcin a suggested intro paragraph which summarises a little about the pgconf this week before going into the detail of the history. I am testing adding in one page anchor link for now to see if it works and if it does will add the others. Please let me know if you're happy with the suggested edits.

File: pgconfeu-2024.mdx (hunk 2 above)
Comment (gulcin) [reply pgconfeu-2024.mdx:48]: Thanks Cezz. 

I would not use "thanks to Xata I am able to once again.." part at all. 

[Extension Ecosystem Summit] 
Not chairing the diversity committee but part of the committee
Also I would not invite for Andrew's call this early. Xata dinner is enough to end it.

File: pgconfeu-2024.mdx [hunk 3]
```diff
@@ -0,0 +1,309 @@
...
+## What do I look forward to at PGConf.EU in Athens? 🔍
```
[the remark this answers is not in the document]
Comment (gulcin) [reply pgconfeu-2024.mdx:49]: LGTM

File: pgconfeu-2024.mdx (hunk 2 above)
Comment (gulcin) [reply pgconfeu-2024.mdx:48]: The name is [Extension Ecosystem Summit]. 

The rest is great.

File: pgconfeu-2024.mdx [hunk 4]
```diff
@@ -0,0 +1,395 @@
+---
+title: 'What do I look forward to at PGConf.EU in Athens?'
```
Comment (alexfrancoeur) [inline pgconfeu-2024.mdx:2]:
```suggestion
title: 'What I look forward to at PGConf.EU in Athens'
```

File: pgconfeu-2024.mdx [hunk 5]
```diff
@@ -0,0 +1,395 @@
...
+description: '14th PostgreSQL Conference in Europe is taking place in Athens, Greece from 22-25 October 2024. In this blog post, I will be sharing what I look forward to at PGConf.EU 2024 including a great keynote, extension ecosystem summit, Xata dinner and more.'
```
Comment (alexfrancoeur) [inline pgconfeu-2024.mdx:3]:
```suggestion
description: 'The 14th PostgreSQL Conference in Europe is taking place in Athens, Greece from 22-25 October 2024. In this blog post, I will be sharing what I look forward to at PGConf.EU 2024 including a great keynote, extension ecosystem summit, Xata dinner and more.'
```

**Review by alexfrancoeur — APPROVED** [review]
Great blog @gulcin! I dropped in a few small suggestions. The title feels a little off to me for some reason being a question.

File: pgconfeu-2024.mdx (hunk 5 above)
Comment (gulcin) [reply pgconfeu-2024.mdx]: Agree with above.

**Pull request merged**